src/composables/RoutingSlip/useCreateRoutingSlipChequePayment.ts

```diff
@@ -65,16 +69,32 @@ export function useCreateRoutingSlipChequePayment () {
     return createRoutingSlipChequePaymentForm.value?.validate()
   }
 
+  const getColumnWidth = computed(() => {
+    return isAmountPaidInUsd.value ? 3 : 4
+  })
+
+  function changeAmountPaidInUsd () {
```

Comment:
no return from this function? looks like changing the existing array. May be better to use in the v-modal of a checkbox?

Reply:
there is an existing watch that takes care for us once we change the chequelist array